Corona-Warn-App/src/main/res/layout/include_interoperability.xml

```diff
@@ -151,37 +154,49 @@
 
             </androidx.constraintlayout.widget.ConstraintLayout>
 
-            <TextView
-                android:id="@+id/country_header_description"
-                style="@style/headline6"
+            <include
+                android:id="@+id/no_countries_default_infoview"
+                layout="@layout/include_interop_no_countries_infoview"
                 android:layout_width="match_parent"
                 android:layout_height="wrap_content"
                 android:layout_marginTop="@dimen/spacing_normal"
-                android:accessibilityHeading="true"
-                android:text="@{countryListTitle}"
-                android:visibility="@{FormatterHelper.formatVisibilityText(countryListTitle)}"
+                android:visibility="@{countryData.empty &amp;&amp; isOnboarding ? View.VISIBLE : View.GONE}"
```

Comment:
What do you think about a method in the FormatterHelper?

Reply:
has the same effect, but I can put it into there to be align with the other calls in this layout